Phylum/Phylum.manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+    "author": "Phylum",
+    "categories": [
+        "Compliance and Security",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "",
+    "helpURL": "https://github.com/SumoLogic/sumologic-public-partner-apps/Phylum/README.md",
```

Comment:
the url should be https://github.com/SumoLogic/sumologic-public-partner-apps/blob/master/Phylum/README.md